### PR TITLE
WIP: records: addition of Zenodo legacy serializers.

### LINKED ISSUE #526: deposit: add deposit json legacy serializer
+ deserialiser afterwards

### PR DESCRIPTION
**Pull request by JavierDelgadoFernandez**
* Adds Zenodo serializers to deposit. (closes #526)

Signed-off-by: Javier Delgado <javier.delgado.fernandez@cern.ch>

### CODE REVIEW

**Comment by JavierDelgadoFernandez** [comment]
### Working on:
- [ ] Deposit tests.
- [ ] Schema differences

**Comment by JavierDelgadoFernandez** [comment]
@lnielsen Updated but still working. Test are not parsing.

File: zenodo/modules/records/serializers/__init__.py [hunk 1]
```diff
@@ -38,32 +38,44 @@
...
+from .loaders import json_marshmallow_translator_loader
+from .translators.legacy_to_zenodo import translate as \
+    translate_legacy_to_zenodo
+from .translators.zenodo_to_legacy import translate as \
+    translate_zenodo_to_legacy
 
 # Serializers
 # ===========
 #: Zenodo JSON serializer version 1.0.0
-json_v1 = JSONSerializer(RecordSchemaJSONV1, replace_refs=True)
```
Comment (lnielsen) [inline zenodo/modules/records/serializers/__init__.py]: What is the reason for removing ``replace_refs=True``?

Comment (JavierDelgadoFernandez) [inline zenodo/modules/records/serializers/__init__.py]: Mea culpa. I was checking some things and I forgot about the ``replace_refs``.

**Comment by JavierDelgadoFernandez** [comment]
@lnielsen It is done except the ``JsonRef`` for the license and the ``record_url``. However, it is waiting for a real testing, it can contain errors in the translation of the objects.

**Pull request closed without merging**